frontend/pages/admin/IntegrationsPage/cards/MdmSettings/AppleMdmPage/components/modals/RenewCertModal/RenewCertModal.tsx

```diff
@@ -118,14 +115,7 @@ const RenewCertModal = ({
                 graphicName="file-pem"
                 message="APNs certificate (.pem)"
                 onFileUpload={onSelectFile}
-                filePreview={
-                  certFile && (
-                    <FileDetails
-                      details={{ name: certFile.name }}
-                      graphicName="file-pem"
-                    />
-                  )
-                }
+                fileDetails={certFile ? { name: certFile.name } : undefined}
```

Comment:
Looks like the graphic name may have been accidentally removed here?

Reply:
Confirmed the code passes line 115 `graphicName` in `<FileDetails/>`

Reply:
Confirmed the UI <img width="760" alt="Screenshot 2024-09-16 at 8 00 29 AM" src="https://github.com/user-attachments/assets/983e7e16-180f-418a-86cf-181850d34bee">